From 2fc84b89e7ed4df65bbb029fd120acb24e5a9d96 Mon Sep 17 00:00:00 2001
From: "Shaik, Kareem M" <kareem.m.shaik@intel.com>
Date: Thu, 6 Oct 2016 16:23:09 +0530
Subject: [PATCH 0337/1240] ASoC: Intel: Skylake: Fix Max DSP MCPS value

Currently Max MCPS is specified as 30M, but it should be 350M.
However actual MCPS is 400M, since module CPS are simulated on
HW and it's peak MCPS is taken into count instead of average cycle
count. So it is limited to 350M to avoid MCPS overshooting.

Change-Id: I40f1a5b8ec66611da52cbfc9365b045015133270
Signed-off-by: Kareem Shaik <kareem.m.shaik@intel.com>
Signed-off-by: Ramesh Babu <ramesh.babu@intel.com>
Reviewed-by: audio_build <audio_build@intel.com>
Reviewed-by: Sinha, Mohit <mohit.sinha@intel.com>
Reviewed-by: Kesapragada, Pardha Saradhi <pardha.saradhi.kesapragada@intel.com>
Reviewed-by: Nc, Shreyas <shreyas.nc@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/skylake/skl-topology.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/skl-topology.c b/sound/soc/intel/skylake/skl-topology.c
index 6f0c95e..493fc29 100644
--- a/sound/soc/intel/skylake/skl-topology.c
+++ b/sound/soc/intel/skylake/skl-topology.c
@@ -4874,7 +4874,7 @@ static void skl_tplg_set_pipe_type(struct skl *skl, struct skl_pipe *pipe)
 }
 
 /* This will be read from topology manifest, currently defined here */
-#define SKL_MAX_MCPS 30000000
+#define SKL_MAX_MCPS 350000000
 #define SKL_FW_MAX_MEM 1000000
 
 /*
-- 
2.7.4

